.github/workflows/modules-zstd.yml

```diff
@@ -47,11 +47,6 @@ jobs:
         run: |
           bazel test -c opt --test_output=errors -- $(bazel query 'filter(".*_dslx_test", kind(rule, //xls/modules/zstd/...))')
 
-      - name: Test ZSTD Module - CC Tests (opt)
```

Comment:
curious why we are removing those?

Reply:
We had more complex test cases in DSLX so eventually we decided to remove this file.
We may add C++ tests back it in future for the sake of measuring performance via scheduled block IR.

Reply:
was it frame_header_test.cc?

Reply:
Actually we removed both of them and move the tests to DSLX.